spyder/plugins/completion/kite/plugin.py

```diff
@@ -21,12 +21,17 @@
 from spyder.utils.programs import run_program
 from spyder.api.completion import SpyderCompletionPlugin
 from spyder.plugins.completion.kite.client import KiteClient
+from spyder.plugins.completion.kite.status import KiteStatus
 
 # Third-party imports
 import psutil
 
 logger = logging.getLogger(__name__)
 
+NOT_INSTALLED = 'Not installed'
+RUNNING = 'Running'
+NOT_RUNNING = 'Not running'
```

Comment:
Let's move these constants to be part of `KiteCompletionPlugin` so we don't have to import them separately. You can place them below `COMPLETION_CLIENT_NAME`.